src/importer/DND5SpellImporter.ts

```diff
@@ -34,34 +34,68 @@ export class DND5SpellImporter {
 
     public importSpellsByUrl = async (url: string): Promise<void> => {
         try {
-            const response: multiSpellImport = await axios.get(url);
-            this.importSpellsByData(response.results);
+            axios.get(url).then(response => {
```

Comment:
Splitte das bitte in private Funktionen. Und außerdem arbeite bitte eher mit await als mit then

Reply:
await hats nicht gebracht, bei dem as ich da mache. Dadurch, dass ich später mit einer Liste abeite, die durch Elemente im Await gefüllt werden hat der irgendwie ne leere Liste übergeben.
Die Funktion ist keine 30 Zeilen lang. Ich finde das etwas üertrieben die zu splitten. Wenn Du das gerne möchtest, mache ich das.

Reply:
es geht nicht um die länge. Sondern um die Tiefe und Komplexität. Wenn await nichts bringt hast du await falsch benutzt!

Reply:
```
const response = await axios.get(url);
...
```